core/storage/leveldb/leveldb.cpp

```diff
@@ -20,12 +20,19 @@ namespace fc::storage {
     if (status.ok()) {
       auto l = std::make_shared<LevelDB>();
       l->db_ = std::unique_ptr<leveldb::DB>(db);
-      return std::move(l); // clang 6.0.1 issue
+      return std::move(l);  // clang 6.0.1 issue
```

Comment:
What kind of issue?

Reply:
`clang-format` created that change, this isn't my code.